scenes/eternal_loom_sokoban/components/spools/spool_yellow.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://ce6ehvjr3vs6q"
+path="res://.godot/imported/spool_yellow.png-ed5c85fb376cee6d61d7c44c160fdd16.ctex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://scenes/eternal_loom_sokoban/components/spools/spool_yellow.png"
```

Comment:
Did @PlayMatters draw these spools?

Reply:
No, these were drawn by Rita Brederson. I wasn't sure if in contracting for Endless OS the rights fell under Endless or Rita. All the assets for the sokoban except for the player was done by Rita Brederson.

Reply:
Should the license be set differently?